man/runc-update.8.md

```diff
@@ -33,7 +33,7 @@ other options are ignored.
 
 # OPTIONS
    --resources value, -r value  path to the file containing the resources to update or '-' to read from the standard input
-   --blkio-weight value         Specifies per cgroup weight, range is from 10 to 1000 (default: 0)
+   --blkio-weight value         Specifies per cgroup weight, range is from 10 to 1000 (0 to use system default)
```

Comment:
This should match the output of `runc --help` (so you have to change that as well). Also I'd make the default text match for all of the options (the line below has a different format).